packages/core/src/lib/input/slider-input.svelte

```diff
@@ -94,6 +99,7 @@ const handlePointerMove = (event: PointerEvent) => {
   if (value !== next) {
     value = next;
     dispatch('input', value);
+    dispatch('slide', value);
```

Comment:
Do we need this non-standard `slide` event if it's the same value as the standard `input`?

Reply:
The problem we're trying to solve here is this:

We want the visualization to update instantaneously when the user drags the slider because this tool exists for fine tuning / instantaneous feedback, but we don't want the visualization to update if the user is entering data within the input until an enter keypress or blur occurs. It's basically how [tweakpane handles events](https://cocopon.github.io/tweakpane/) for numeric inputs. I think this may be the best way to go for real-time feedback, and there are some alternatives that I've tried that seem to have more problems, like updating on every input event but having exceptions for trailing decimals or for zeros after decimals so that they don't get sheared off in controlled inputs. Curious if you have thoughts or ideas!

I also want to probably have a guild meeting in the near future to voice some of these problems and begin to formalize solutions

Reply:
That sounds like the difference between `input` (value changes instantaneously) and `change` (user finished entering a value). My gut instinct would be to treat this more like a vanilla `input[type="range"]`

Playing around with Tweakpane, this seems to match their behavior, in that the position of the slider - the instantaneous value - is not moved until the text box value is confirmed via Enter or blur.

- Fire `input` whenever the slider is moved
- Fire `change` whenever the slider is released
- **Do not fire `input`** on textbox `input`, keep it internal
- Fire `input` + `change` on textbox `change`

Reply:
^ I like this approach a lot. I'm going to turn this description into a ticket, since I think this is something we'll need to ensure that we're doing across the board